src/app/code/community/Zendesk/Zendesk/Helper/Data.php

```diff
@@ -188,4 +268,230 @@ public function isExternalIdEnabled()
     {
         return Mage::getStoreConfig('zendesk/general/use_external_id');
     }
+    
+    /**
+     * Retrieve ticket statistics
+     */
+    public function getTicketTotals($type = null, $from = null, $to = null)
+    {
+        try {
+        $tickets = Mage::getModel('zendesk/api_tickets')->all();
+        }
+        catch(Exception $ex) {
+            return array();
+        }
+        
+        if( is_null($tickets) )
+        {
+            return false;
+        }
+        
+        $totals = array(
+            'open'      =>  0,
+            'new'       =>  0,
+            'solved'    =>  0,
+            'closed'    =>  0,
+            'all'       =>  0,
+            'pending'   =>  0
+        );
+
+        if( $from )
+            $from = strtotime($from);
+        else
+            $from = 0;
+
+        foreach( $tickets as $ticket )
+        {
+            if( $from || $to )
+            {
+                if( strtotime($ticket['created_at']) > $from)
+                {
+                    if( $to )
+                    {
+                        if( strtotime($ticket['created_at']) < strtotime($to) )
+                        {
+                            if( isset($totals[$ticket['status']]) )
+                                $totals[$ticket['status']]++;
+                            
+                            $totals['all']++;
+                        }
+                    }
+                    else
+                    {
+                        if( isset($totals[$ticket['status']]) )
+                            $totals[$ticket['status']]++;
+                        
+                        $totals['all']++;
+                    }
+                }
+            }
+            else
+            {
+                if( isset($totals[$ticket['status']]) )
+                    $totals[$ticket['status']]++;
+                    
+                $totals['all']++;
+            }
+        }
+        
+        
+        if( $type && isset($totals[$type]))
+        {
+            return $totals[$type];
+        }
+        else
+        {
+            return $totals;
+        }
+    }
+    
+    public function getExcerpt($row)
```

Comment:
Tickets which are created from contact us form don't have subject. Title is a direct link to the ticket in zendesk, if it will be blank there will not be any way to go to this ticket. Do you really want to leave it blank or maybe set to "No Subject"?